Lib/test/test_tools/test_lll.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for the lll script in the Tools/script directory."""
+
+import os
+import sys
+import tempfile
+from test import support
+from test.test_tools import skip_if_missing, import_tool
+import unittest
+
+skip_if_missing()
+
+
+class lllTests(unittest.TestCase):
+
+    def setUp(self):
+        self.lll = import_tool('lll')
+        oldargv = sys.argv
+
+        def fixup():
+            sys.argv = oldargv
+        self.addCleanup(fixup)
+
+    def test_lll_multiple_dirs(self):
+        with tempfile.TemporaryDirectory() as dir1, \
+             tempfile.TemporaryDirectory() as dir2:
+            fn1 = os.path.join(dir1, 'foo1')
+            fn2 = os.path.join(dir2, 'foo2')
+            for fn, dir in (fn1, dir1), (fn2, dir2):
+                open(fn, 'w').close()
+                os.symlink(fn, os.path.join(dir, 'symlink'))
+
+            sys.argv = ['lll', dir1, dir2]
+            with support.captured_stdout() as output:
+                self.lll.main()
+            self.assertEqual(output.getvalue(),
+                f'{dir1}:\n'
+                f'symlink -> {fn1}\n'
+                '\n'
```

Comment:
```suggestion
                f'\n'
```
Or you can use triple quotes to embed newlines.